### PR TITLE
LRQA-72313

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://issues.liferay.com/browse/LRQA-72313

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:auditing

**Review by zhangyan0701 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase [hunk 1]
```diff
@@ -26,6 +26,78 @@ definition {
...
+		User.firstLoginUI(
+			password = "test",
+			userEmailAddress = "test@liferay.com");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Security",
+			panel = "Control Panel",
+			portlet = "Audit");
+
+		AssertElementNotPresent(locator1 = "AuditReports#REPORTS_TABLE_RESOURCE_ACTION");
```
[the remark this answers is not in the document]
Comment (catarineaguiar) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase]: I used "AuditReports#REPORTS_TABLE_RESOURCE_ACTION_1" but it is hided in the page, then I used this one: "AuditReports#REPORTS_DETAIL_RESOURCE_NAME" and worked.

[the remark this answers is not in the document]
Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase]: "AuditReports#REPORTS_DETAIL_RESOURCE_NAME" this one is in the report details page, not in the report table. But in this step, we are checking that the report table should not contain any audit results, right? So I don't think we should use this path.

I tried "AuditReports#REPORTS_TABLE_RESOURCE_ACTION_1" again and running the test locally, and it worked for me, the test passed. Could you try with this one again?

[the remark this answers is not in the document]
Comment (catarineaguiar) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase]: I don't know why here this path fails.

```
 [exec]     BUILD FAILED: Element is present at "xpath=(//tr/td[5])[1]"
     [exec]     /home/me/dev/projects/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase[CanBeDisabledToPersistInDBThroughSystemSettingsAndOSGIConfigFile]:59
     [exec] 
     [exec] 
     [exec] com.liferay.poshi.runner.PoshiRunner > test[LocalFile.AuditConfiguration#CanBeDisabledToPersistInDBThroughSystemSettingsAndOSGIConfigFile] STANDARD_ERROR
     [exec]     java.lang.Exception: Element is present at "xpath=(//tr/td[5])[1]"
     [exec]     	at com.liferay.poshi.runner.selenium.BaseWebDriverImpl$Condition.assertTrue(BaseWebDriverImpl.java:4506)
     [exec]     	at com.liferay.poshi.runner.selenium.BaseWebDriverImpl$Condition.waitFor(BaseWebDriverImpl.java:4541)
     [exec]     	at com.liferay.poshi.runner.selenium.BaseWebDriverImpl.waitForElementNotPresent(BaseWebDriverImpl.java:3226)
     [exec]     	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor$1.call(PoshiRunnerExecutor.java:1123)
     [exec]     	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
     [exec]     	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
     [exec]     	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
     [exec]     	at java.lang.Thread.run(Thread.java:748)
```

**Comment by catarineaguiar** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase (hunk 1 above)
[the remark this answers is not in the document]
Comment (catarineaguiar) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase]: I used a macro to assert the Alert.

[the remark this answers is not in the document]
Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/cpauditreports/usecase/AuditConfiguration.testcase]: Yes, you are right, "AuditReports#REPORTS_TABLE_RESOURCE_ACTION_1" is hidden in the page. It's weird that it worked for me before. 


> I used a macro to assert the alert message.

Cool, let's do this way. This should be better, I just think the same.

**Pull request closed without merging**